LiskHQ/lisky
# Add node install command - Closes #684

### What was the problem?

### How did I fix it?
`node:install` command implementation, will support the installation of `lisk` software for Testnet (Test Network) or the Mainnet (Main Network) from the official [download website](https://downloads.lisk.io), and install it on the local machine.
### How to test it?

- git clone commander
- npm i
- `./bin/run node:install --no-snapshot --network=testnet --name=testnet-1.5.0`
- `ls -l ~/.lisk/network` you should see `testnet-1.5.0` folder with lisk software

### Review checklist

* The PR resolves #684 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
